### ISSUE TITLE
When answering Likert Questions- Change error for no answer

### ISSUE THREAD

**Issue by djmoffat** [BrechtDeMan/WebAudioEvaluationTool#130]
When you leave a likert question blank, you get the error "You have not moved tracks x" - should be "have not answered" or something similar.

**Comment by nickjillings** [comment]
I can't replicate this issue. By a question do you mean the survey, on page comment or a comment-question box. If comment-question, which type?

**Comment by djmoffat** [comment]
I mean making likert and changing scale

Basically when `<interfaceoption type="check" name="fragmentMoved"/>` is applied it gives an odd error for likert

**Comment by nickjillings** [comment]
If you select a likert button before you actually listen to that fragment, it doesn't count it as a move. Is that what is happening?

**Comment by nickjillings** [comment]
Ah OK, sorry, got you now!!

**Comment by djmoffat** [comment]
let me rephrase:

Is it possible to change the error message attached to an `<interfaceoption>`
